test/test_serialization.py

```diff
@@ -588,17 +588,19 @@ class TestBothSerialization(TestCase, SerializationMixin):
     def test_serialization_new_format_old_format_compat(self):
         x = [torch.ones(200, 200) for i in range(30)]
 
-        def test(filename):
-            torch.save(x, filename, _use_new_zipfile_serialization=True)
-            x_new_load = torch.load(filename)
+        def test(f_new, f_old):
+            torch.save(x, f_new, _use_new_zipfile_serialization=True)
+            f_new.seek(0)
+            x_new_load = torch.load(f_new)
```

Comment:
Not sure, what is being tested here, but save/load from file descriptor and save/load to/from filename are not equivalent and can contain separate bugs